Week1/homework/js2-exercises/2-aboutMe.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>About Me</title>
+  </head>
+  <body>
+    <h1>About Me</h1>
+
+    <ul>
+      <li>Nickname: <span id="nickname"></span></li>
+      <li>Favorite food: <span id="fav-food"></span></li>
+      <li>Hometown: <span id="hometown"></span></li>
+    </ul>   
+	<script>
+		document.body.style.fontFamily=`Arial,sans-serif`;
+
+		document.querySelector('#nickname').innerText='David';
+		document.querySelector('#fav-food').innerText='Charcoal grilled meat';
+		document.querySelector('#hometown').innerText='London';
+
+		const items=document.querySelectorAll('ul li');
+		items.forEach(item=> {item.classList.add('list-item')})
+
+		const styleTag=document.createElement('style');
+		document.querySelector("head").appendChild(styleTag);
+		document.querySelector('style').innerText= '.list-item {color:red}'
```

Comment:
I believe you misunderstood the requirement here or got carried away manipulating using javascript 😄 
![image](https://user-images.githubusercontent.com/36904389/71363075-37399a00-2598-11ea-9033-47d734d387a5.png)
The idea is to add a tag in the html file and have a css file with the class item-list which sets font color to red.